src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -495,7 +496,7 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
 }
 
 export const ShippingFragmentContainer = createFragmentContainer(
-  injectCommitMutation(injectDialog(ShippingRoute)),
+  withSystemContext(injectCommitMutation(injectDialog(ShippingRoute))),
```

Comment:
helper that creates the HOC that adds `SystemContextProps` which includes `relayEnvironment`

Reply:
I was going to has what that was for!